src/profile-logic/call-tree.js

```diff
@@ -61,7 +61,7 @@ function extractFaviconFromLibname(libname: string): string | null {
       'Error while extracing the favicon from the libname',
       libname
     );
-    return null;
+    return '';
```

Comment:
Ah, sorry about that, I think this is better with ` null` here after all... Sometimes we can think things but when looking at the full picture we realize this isn't a good idea...

Reply:
Yea, I understand